packages/design-system/src/components/ChoiceList/Choice.stories.tsx

```diff
@@ -13,17 +14,9 @@ export default {
     errorMessage: 'You must agree to the terms and conditions before continuing',
     defaultChecked: false,
   },
-  argTypes: {
-    checkedChildren: { control: 'text' },
-    errorMessage: { control: 'text' },
-    hint: { control: 'text' },
-    label: { control: 'text' },
-    requirementLabel: { control: 'text' },
-    uncheckedChildren: { control: 'text' },
-    value: { control: 'text' },
-  },
```

Comment:
Some of the controls are wonky on the docs page, but overall this did a nice job of inferring the types. And the `ReactNode` type is way more usable in this version!

Reply:
No action needed here. Just commenting.